### PR TITLE
Remove test/Project.toml

### PR DESCRIPTION
**Pull request by odow**
Found by https://github.com/jump-dev/SolverTests/actions/runs/4715567163/jobs/8362641484

Rational issue is fixed on the upcoming MOI v1.15

### CODE REVIEW

**Comment by odow** [comment]
Hmm. This still failed locally. I need to take a deeper look

**Comment by odow** [comment]
If you include an extras section in `Project.toml`, then it's just confusing to have `test/Project.toml`.

File: test/MOI_wrapper.jl [hunk 1]
```diff
@@ -43,6 +43,7 @@ using CDDLib
                     # TODO Should be fixed in MOI master
                     "test_model_LowerBoundAlreadySet",
                     "test_model_UpperBoundAlreadySet",
+                    "test_linear_open_intervals",
```
Comment (blegat) [inline test/MOI_wrapper.jl:46]: It should be excluded in the `else` clause. But it should work with MOI master now

**Comment by odow** [comment]
Starting again because you merged #83

**Pull request closed without merging**